5_databases/week1/Homework.sql

```diff
@@ -0,0 +1,57 @@
+select * FROM task;
+SELECT * from status;
+select * FROM user;
+
+1--Find out how many tasks are in the task table
+
+SELECT COUNT(id) from `task`;
+
+2--Find out how many tasks in the task table do not have a valid due date
+
+SELECT COUNT(id) FROM `task` WHERE due_date IS  NULL;
+
+3--Find all the tasks that are marked as done
+
+SELECT task.id,
+       status.name
```

Comment:
When you get the results of this query, the columns will be named `id` and `name`. This is a bit confusing, what could be done differently? 🤔